ocs_ci/utility/performance_dashboard.py

```diff
@@ -0,0 +1,98 @@
+"""
+Script to save result data using json into the performance dashboard
+Database structure:
+commitid: The specific buid id. : v4.3.0-407 E.g. 407
+project: The project we are currently testing. E.g. 4.3
+branch: OCS full version. E.g. 4.3.0
+executable: Which OCS version we are testing. E.g. 4.3
+benchmark: The benchmark type (based on interface)
+environment: The platform we are testing. E.g, AWS
+result_value: The value of this benchmark.
+
+"""
+import requests
+import json
+from datetime import datetime
+from ocs_ci.framework import config
+from ocs_ci.ocs import ocp, defaults
+from ocs_ci.ocs.node import get_typed_nodes
+from ocs_ci.ocs import constants
+
+current_date = datetime.today()
+URL = 'http://10.0.78.167:8000/'
+
+data_template = {
+    "commitid": None,
+    "project": None,
+    "branch": None,
+    "executable": None,
+    "benchmark": None,
+    "environment": None,
+    "result_value": None
+}
+
+
+def initialize_data():
+    """
+    Initialize the data dictionary with cluster data
+    """
+    csv = ocp.OCP(kind='csv', namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    worker_type = get_typed_nodes(num_of_nodes=1)[0].data['metadata'][
+        'labels'
+    ]['beta.kubernetes.io/instance-type']
+
+    csv_vers = csv.get()['items'][1]['spec']['version'][:-3].split("-")
```

Comment:
it seems we have isseu with this part of the code 
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-test-pr/1705/testReport/junit/tests.e2e.performance.test_io_performance/TestIOPerformance/test_run_io_RBD_1GB_rw_6_60_16_/